sysig.py

```diff
@@ -33,7 +50,52 @@ def get_cpu_util():
         cpu_val = psutil.cpu_percent(interval=1, percpu=False)
         dpg.set_value(cpu_progress_bar, 1.0 / 100.0 * cpu_val)
         dpg.configure_item(cpu_progress_bar, overlay=f"{cpu_val}%")
-
+        
+# Get CPU Temperature
+def get_cpu_temperature():
+    if platform.system() == 'Windows':
+        try:
+            # Try using psutil first
+            temps = psutil.sensors_temperatures()
+            if 'coretemp' in temps:
+                return temps['coretemp'][0].current
+        except Exception as e:
+            pass
+
+        try:
+            # Fallback to WMI approach if psutil does not provide the info
+            import wmi
+            w = wmi.WMI(namespace="root\\wmi")
+            temperature_info = w.MSAcpi_ThermalZoneTemperature()[0]
+            return temperature_info.CurrentTemperature / 10.0 - 273.15
+        except wmi.x_access_denied:
+            return "Access Denied (Run script as administrator)"
```

Comment:
Is there any solution for this without using administrator access in Windows? All of the modules I used don't need administrator access.

Reply:
That was the problem I was running into, I researched the problem for a while and the solutions were to run wmi using admin access, or use additional software like OHM.  As the graphics additions are working, I can scrap the cpu temp idea until I can figure out a better solution for Windows / Intel.  I can remove the cpu temperature code for now if you would like to incorporate just the gpu additions.

Reply:
> use additional software like OHM.

Please don't, This is just a simple app 😅

> I can remove the cpu temperature code for now if you would like to incorporate just the gpu additions.

If it's okay with you, Please do. It's better to do GPU additions for now, and you can submit another PR if you have a solution for admin access.

Reply:
lol yea I like that it is a simple app.  I was trying to implement an easy way to see cpu temp but it turned out to be a rabbit hole lol.  I like keeping it simple.  That sounds good.  I will edit the code and send that in a moment

Reply:
Thanks! I will run the workflow and merge this later.